mopidy_gmusic/library.py

```diff
@@ -205,6 +205,11 @@ def browse(self, uri):
 
         return []
 
+    def get_images(self, uris):
+        logger.debug("Looking up images: %s", uris)
```

Comment:
This might be a bit more readable:

```suggestion
        logger.debug(f"Looking up images: {uris}")
```